src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.Configuration/Azure.Configuration.csproj

```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Description>This is the implementation of the Azure SDK for Azure Configuration</Description>
-    <AssemblyTitle>Microsoft Azure Configuration</AssemblyTitle>
+    <Description>This is the implementation of the Azure SDK for Azure App Configuration Service</Description>
```

Comment:
The whole product we create is called Azure SDK. Each dll/package in it is called "Azure SDK Component", and so I would reword it to: Azure SDK Component for Azure App Configuration Service.